### PR TITLE
docs: update toolbar usage component

### PR DESCRIPTION
**Pull request by KareemDa**
## Description

- add  `outlined`, `rounded`, `shaped` to tabs
- add `elevation` to sliders.

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by KareemDa** [comment]
@MajesticPotatoe 🙋

**Comment by KareemDa** [comment]
@MajesticPotatoe is there any problems with this PR to be merged ?

File: packages/docs/src/pages/en/components/toolbars.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ meta:
 
 # Toolbars
 
-The `v-toolbar` component is pivotal to any gui, as it generally is the primary source of site navigation. The toolbar component works great in conjunction with [v-navigation-drawer](/components/navigation-drawers) and [v-card](/components/cards).
+The `v-toolbar` component is pivotal to any graphical user interface (GUI)  , as it generally is the primary source of site navigation. The toolbar component works great in conjunction with [v-navigation-drawer](/components/navigation-drawers) and [v-card](/components/cards).
```
Comment (MajesticPotatoe) [inline packages/docs/src/pages/en/components/toolbars.md:15]:
```suggestion
The `v-toolbar` component is pivotal to any graphical user interface (GUI), as it generally is the primary source of site navigation. The toolbar component works great in conjunction with [v-navigation-drawer](/components/navigation-drawers) and [v-card](/components/cards).
```

**Review by MajesticPotatoe — APPROVED** [review]

**Comment by MajesticPotatoe** [comment]
> @MajesticPotatoe is there any problems with this PR to be merged ?
minor fix, already done.

FYI. There's no need to ping people for review of PR's (especially when they were just recently created). We already receive notification when are created and will review them at our earliest convenience.

**Comment by KareemDa** [comment]
> > @MajesticPotatoe is there any problems with this PR to be merged ?
> > minor fix, already done.
> 
> FYI. There's no need to ping people for review of PR's (especially when they were just recently created). We already receive notification when are created and will review them at our earliest convenience.

Roger that

**Pull request merged**